tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,251 @@
+"""CLI tests for logging.
+
+:Requirement: Logging
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+from fauxfactory import gen_string
+from robottelo import ssh
+from robottelo.ssh import upload_file
+from robottelo import manifests
+from robottelo.test import CLITestCase
```

Comment:
Import should be in order.

Reply:
Hello @omkarkhatavkar 
isort suggests this:

```
import re

from fauxfactory import gen_string

from robottelo import manifests, ssh
from robottelo.cli.factory import make_org, make_product_wait, make_repository
from robottelo.cli.subscription import Subscription
from robottelo.ssh import upload_file
from robottelo.test import CLITestCase
```

or do you prefer:

```
import re
from fauxfactory import gen_string
from robottelo import (
    manifests,
    ssh,
)
from robottelo.cli.factory import (
    make_org,
    make_product_wait,
    make_repository,
)
from robottelo.cli.subscription import Subscription
from robottelo.ssh import upload_file
from robottelo.test import CLITestCase

```

Reply:
both look good, any of them is fine

Reply:
OK, using 2nd format as it seems commonly used.
Thank you